osmosis/assetlist.json

```diff
@@ -14261,6 +14261,26 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/bOSMO.png"
         }
       ]
+    },
+    {
+      "description": "Memecoin for The International Brane Wave",
+      "denom_units": [
+        {
+          "denom": "factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz",
+          "exponent": 0
+        },
+        {
+          "denom": "brnz",
+          "exponent": 0
+        }
+      ],
+      "base": "factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz",
+      "name": "BRNZ",
+      "display": "brnz",
```

Comment:
It is best to have display refer to the denom of a unit rather than an alias, so please change display value to factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz.
Note that interface will only be displaying the name and symbol "BRNZ"--'display' is only used for informing which exponent to use.